.cirrus.yml

```diff
@@ -101,6 +101,26 @@ fedora_packaging_task:
 
     timeout_in: '20m'
 
+# Verify calls to bin/config were saved
+config_task:
+    env:
+        GOSRC: $CIRRUS_WORKING_DIR
+    # Runs within Cirrus's "community cluster"
+    container:
+        matrix:
+               # fedora:28 doesn't have go mod by default
+               # and we should only need one check to make sure
+               # config changes were synced
+            image: "registry.fedoraproject.org/fedora:29"
+        cpu: 4
+        memory: 12
+
+    script:
+        - dnf install -y make glib2-devel git gcc rpm-build golang
+        - cd $CIRRUS_WORKING_DIR
+        - GO111MODULE=on go mod init github.com/containers/conmon
+        - make config
+        - ./hack/tree_status.sh
```

Comment:
My preference would be to have these commands inside something like `contrib/cirrus/config_verify.sh`.  Not only to cut down the number of lines in the YAML (it's quite big), but also because then running the script manually becomes possible if/when needed.